Tests/SwiftParserTest/RegexLiteralTests.swift

```diff
@@ -495,8 +553,37 @@ final class RegexLiteralTests: ParserTestCase {
       /1️⃣  /
       """,
       diagnostics: [
-        DiagnosticSpec(message: "bare slash regex literal may not start with space")
-      ]
+        DiagnosticSpec(
+          message: "bare slash regex literal may not start with space",
+          fixIts: [
+            "convert to extended regex literal",
+            #"insert '\'"#,
+          ]
+        )
+      ],
+      applyFixIts: ["convert to extended regex literal"],
+      fixedSource: """
+        #/  /#
+        """
+    )
+
+    assertParse(
+      """
+      /1️⃣  /
+      """,
+      diagnostics: [
+        DiagnosticSpec(
+          message: "bare slash regex literal may not start with space",
+          fixIts: [
+            "convert to extended regex literal",
+            #"insert '\'"#,
+          ]
+        )
+      ],
+      applyFixIts: [#"insert '\'"#],
+      fixedSource: #"""
+        /\ \ /
```

Comment:
Could you also add a test case where the regex contains more than 2 spaces. Just to make sure that we only escape the first and last and not the ones in between. (or just make `testForwardSlashRegex164` apply the `insert '\'` Fix-It.